main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/go-playground/webhooks.v5/github"
+	"net/http"
+)
+
+const (
+	path = "/payload"
+)
+
+func main() {
+	hook, _ := github.New(github.Options.Secret("<GITHUBSECRET>"))
+
+	http.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+		payload, err := hook.Parse(r, github.PushEvent)
+		if err != nil {
+			if err == github.ErrEventNotFound {
+				// ok event wasn't one of the ones asked to be parsed
+				println("Error receiving github payload")
+				fmt.Println(err)
+
+			}
```

Comment:
You’re silently ignoring errors that aren’t of the type you expect! Also if you get an error, you’re falling thru and operating on a nil value — probably not what you want!

Reply:
True, will update